### PR TITLE
security/clamav: add third party signatures

### LINKED ISSUE #1162: [Feature Request] ClamAV - Unofficial free signatures updates to improve the detection rate?
Since ClamAV is not one of the best virus scanners I have read that there are sources that provide regular updates with better signatures and thus would significantly improve the recognition rate of ClamAV.

I have collected a few links to the best of my knowledge and belief, which could perhaps improve opnsense or the clamav virus protection. Maybe someone can take a closer look at the pages if he has time to guarantee that it could improve ClamAV reputably and in the long run. Unfortunately, I am not so well versed in the signatures of a virus scanner and can not judge that well if that would really pay off.

This is just an idea !. What do you think?

https://sanesecurity.com/
https://malware.expert/signatures/

### PR DESCRIPTION
**Pull request by mimugmail**
Closes #1162

### CODE REVIEW

**Comment by opnsenseuser** [comment]
double 👍 -> i think one of the best improvements!! please, please merge!!!

**Comment by opnsenseuser** [comment]
grafik

**Comment by opnsenseuser** [comment]
On their page, mailexpert has described the instructions for the case of the false positives and what or how to solve them. do you think you can still build this into the current Version?


Malware.Expert.Generic.Eval.1
--

**Comment by opnsenseuser** [comment]
Lets make it Perfect 🥇

**Comment by opnsenseuser** [comment]
Mailexperts and sanesecurity use the same two files for whitelisting. The files have to be created in the clamav Database folder.

**Comment by opnsenseuser** [comment]
It would be best to have a selection box where the user could simply add the selected false positives via the "add to whitelist" Button. that would make it perfect. The feature does not even have pfsense in the clamav.

**Comment by mimugmail** [comment]
You mean a checkbox? This would mean all content of a file has to be loaded into the model to allow this. I'd not recommend this as the hoster of such a file could easily crash your config.xml when adding some megabytes of data.

**Comment by opnsenseuser** [comment]
@mimugmail what would you suggest?

**Comment by mimugmail** [comment]
merge to master, keep in dev, see how many FP's are coming. Then we'll see if it's worth the work to add a ignore for each file.

**Comment by opnsenseuser** [comment]
Ok 👍

File: security/clamav/pkg-descr [hunk 1]
```diff
@@ -5,4 +5,39 @@ including a flexible and scalable multi-threaded daemon,
...
+
+* New tab to show current ClamAV version
+
+1.1
+
+* Make signatures persistant in MFS installations
+
+1.0
+
+* Initial release
```
Comment (fichtner) [inline security/clamav/pkg-descr]: zap

File: security/clamav/pkg-descr [hunk 2]
```diff
@@ -5,4 +5,39 @@ including a flexible and scalable multi-threaded daemon,
...
+
+* Add Log viewer
+
+1.2
+
+* New tab to show current ClamAV version
+
+1.1
+
+* Make signatures persistant in MFS installations
```
Comment (fichtner) [inline security/clamav/pkg-descr]:
```suggestion
* Make signatures persistent in MFS installations
```

File: security/clamav/pkg-descr [hunk 3]
```diff
@@ -5,4 +5,39 @@ including a flexible and scalable multi-threaded daemon,
...
+================
+
+1.6
+
+* Add optional signature sources
+
+1.5
+
+* Fix symlink permission
+* Fixed some bugs in Log viewer
```
Comment (fichtner) [inline security/clamav/pkg-descr]:
```suggestion
* Fixed some bugs in log viewer
```

**Comment by mimugmail** [comment]
Done, no idea why I always write persistent wrong :)

**Comment by fichtner** [comment]
Merged, thanks!

**Comment by fichtner** [comment]
It's a persistent issue for sure. :D

**Comment by opnsenseuser** [comment]
Just for your information! (Forum topic -tests)
https://forum.opnsense.org/index.php?topic=11629

**Pull request merged**